src/SFA.DAS.AssessorService.Application.Api.UnitTests/Validators/ExternalApi/Learners/GetBatchLearnerRequestValidator/WhenMissingFamilyName.cs

```diff
@@ -0,0 +1,34 @@
+﻿using FizzWare.NBuilder;
+using FluentAssertions;
+using FluentValidation.Results;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models.ExternalApi.Learners;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Validators.ExternalApi.Learners.GetBatchLearnerRequestValidator
+{
+    public class WhenMissingFamilyName : GetBatchLearnerRequestValidatorTestBase
+    {
+        private ValidationResult _validationResult;
+
+        [SetUp]
+        public async Task Arrange()
+        {
+            var request = Builder<GetBatchLearnerRequest>.CreateNew()
+                .With(i => i.Uln = 1234567890)
```

Comment:
Maybe there could be constants in the base class which define which Uln are valid or invalid and then they are used here to trigger the validation errors - that would be very clear then which was which

Reply:
This will get done under Tech Debt ticket ON-2130 when all the validators are refactored